### ISSUE TITLE
Slack Integration

### ISSUE THREAD

**Issue by Bleez103** [neiljohn23/SupMonitor#1]
Is it possible to add Slack to your monitor?

**Comment by neiljohn23** [comment]
Explain what you mean by add slack to the monitor. Like have notifications be sent through slack?

**Comment by Bleez103** [comment]
Yes. Have notifications sent when a restock occurs.. similar how the twitter integration works.
